### ISSUE TITLE
How do you download Kivy on Android plz

### ISSUE THREAD

**Issue by Minecraft-Gamer-Nation** [kivy/kivy-launcher#29]

**Comment by chrysn** [comment]
This is a duplicate of https://github.com/kivy/kivy-launcher/issues/14

**Comment by pedroavex** [comment]
@chrysn i did rhe steps you mentioned for downloading the package (apk) from the last build on "Actions" page. However when i install the APK on my mobile it makes Kaspersky very angry showing a trojan threat related to remote access. Do you have idea if this is a false alarm for some specific reason? I got afraid and didn't run it... Thanks!

**Comment by chrysn** [comment]
This is unrelated to the means of download; please either file this as a separate issue to kivy-launcher, or complain to your anti-virus vendor directly.

**Comment by pedroavex** [comment]
Yes you are absolutely right, i should have filed as a separate issue, my bad. Sorry for that. Thanks!
